sbi/neural_nets/estimators/flowmatching_estimator.py

```diff
@@ -293,10 +295,12 @@ def diffusion_fn(self, input: Tensor, times: Tensor) -> Tensor:
             Diffusion function at a given time.
         """
         # analytical g(t) is undefined at t = 1.
+        # NOTE: We bound the singularity to avoid numerical issues i.e. 1 - t > 0.01
+        # this effectively prevents and explosion of the SDE in the beginning.
         return torch.sqrt(
             2
             * (times + self.noise_scale)
-            / torch.maximum(1 - times, torch.tensor(1e-6).to(times))
+            / torch.maximum(1 - times, torch.tensor(1e-2).to(times))
```

Comment:
would it make sense to move this into a function argument and add the explanation in the docstring? same for the change above in `drift_fn`?

Reply:
Yes, we can do that. The actual values isn't that important it just should be smaller.

Effectively, you can also avoid this by reducing "t_max" from 1. to 0.99.

Reply:
in any case, I suggest adding this as an arg with docstring.